src/oom/oomd-manager.c

```diff
@@ -378,17 +378,29 @@ static int monitor_cgroup_contexts_handler(sd_event_source *s, uint64_t usec, vo
                         OomdCGroupContext *t;
 
                         SET_FOREACH(t, targets) {
-                                log_notice("Memory pressure for %s is greater than %lu for more than %"PRIu64" seconds and there was reclaim activity",
-                                        t->path, LOAD_INT(t->mem_pressure_limit), m->default_mem_pressure_duration_usec / USEC_PER_SEC);
+                                _cleanup_free_ char *selected = NULL;
 
-                                r = oomd_kill_by_pgscan_rate(m->monitored_mem_pressure_cgroup_contexts_candidates, t->path, m->dry_run);
+                                log_debug("Memory pressure for %s is %lu.%02lu%% which is greater than "
+                                           "%lu.%02lu%% for more than %"PRIu64" seconds and there was reclaim activity",
```

Comment:
Maybe "is ... which is greater than ... for more than ... seconds" → "is ... (>...) for >... s",
"there was reclaim activity" → "reclaim activity seen" ?
Very long lines are a bit annoying to read.